api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -479,6 +479,15 @@ type RouteTableIDRange struct {
 
 type RouteTableRanges []RouteTableIDRange
 
+func (r RouteTableRanges) Len() int {
```

Comment:
Sort of a nit, but `Len()` is perhaps a confusing name for this since it's a method on a slice, and normally Len() of a slice is the number of entries in that slice, whereas this is a more complex calculation.

Perhaps "NumAvailableRouteTables" or something like that?

Reply:
Good call, will amend